lib/LedgerAgent.js

```diff
@@ -11,10 +11,11 @@ const routes = config['ledger-agent'].routes;
 module.exports = class LedgerAgent {
   constructor(options) {
     this.id = options.id;
-    this.node = options.node;
+    this.ledgerNode = options.ledgerNode;
     this.owner = options.owner;
     this.name = options.name;
     this.description = options.description;
+    this.peerId = options.peerId;
```

Comment:
Let's expose this as `targetNode` and change in ledger agent internally. From a "ledger agent perspective" the "peer ID" it gets from consensus should be its `targetNode`.

Reply:
@dlongley sounds fine.